CMakeLists.txt

```diff
@@ -80,18 +79,23 @@ if (ENABLE_OPENMP)
 endif()
 if (ENABLE_CUDA)
   list(APPEND RAJA_PERFSUITE_DEPENDS cuda)
-endif() 
-if (ENABLE_HIP)
+endif()
+# Kokkos requires hipcc as the CMAKE_CXX_COMPILER for HIP AMD/VEGA GPU
+# platforms, whereas RAJAPerf Suite uses blt/CMake FindHIP to set HIP compiler 
+# Separate RAJAPerf Suite and Kokkos handling of HIP compilers
+if ((ENABLE_HIP) AND (NOT ENABLE_KOKKOS))
+#if (ENABLE_HIP)
   list(APPEND RAJA_PERFSUITE_DEPENDS hip)
 endif()
 
 set(RAJAPERF_BUILD_SYSTYPE $ENV{SYS_TYPE})
 set(RAJAPERF_BUILD_HOST $ENV{HOSTNAME})
 
 if (ENABLE_CUDA)
-  set(CMAKE_CUDA_STANDARD 11)
+  if (ENABLE_CUDA AND ENABLE_KOKKOS)
```

Comment:
The ENABLE_CUDA here is redundant (since we are already inside an `if (ENABLE_CUDA)` block.